bot/exts/info/tags.py

```diff
@@ -145,8 +145,7 @@ async def _send_matching_tags(self, ctx: Context, keywords: str, matching_tags:
         else:
             is_plural = keywords.strip().count(' ') > 0 or keywords.strip().count(',') > 0
             embed = Embed(
-                title=f"Here are the tags containing the given keyword{'s' * is_plural}:",
-                description='\n'.join(tag['title'] for tag in matching_tags[:10])
+                title=f"Here are the tags containing the given keyword{'s' * is_plural}:"
```

Comment:
You say that you've 'made some fixes there', could you expand on that? What is the reason for the diff here?

Reply:
`description` field is ignored by the paginator.

Reply:
Okay, thanks, that makes sense. I'll try to do a full review soon.